CHANGELOG.md

```diff
@@ -69,7 +69,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 ### State Machine Breaking
 
 * (light-clients/07-tendermint) [\#2554](https://github.com/cosmos/ibc-go/pull/2554) Forbid negative values for `TrustingPeriod`, `UnbondingPeriod` and `MaxClockDrift` (as specified in ICS-07).
-* (06-solomachine) [\#2744](https://github.com/cosmos/ibc-go/pull/2744)  `Misbehaviour.ValidateBasic()` now only enforces that signature data does not match when the signature paths are different.
+* (06-solomachine) [\#2744](https://github.com/cosmos/ibc-go/pull/2744) `Misbehaviour.ValidateBasic()` now only enforces that signature data does not match when the signature paths are different.
+* (06-solomachine) [\#2748](https://github.com/cosmos/ibc-go/pull/2748) Adding sentinel value for header path in 06-solomachine.
```

Comment:
let's add these to the description of the PR too in conventional format! ❤️

Reply:
Is this correct?

`chore(statemachine)!: Adding sentinel value for header path in 06-solomachine.`

Reply:
yeah that looks good!